app/routes/chatbot.js

```diff
@@ -320,9 +320,9 @@ router.post('/', (req, res) => {
     })
     .catch(err => {
       if (err.message === 'broadcast declined') {
-        const msg = scope.broadcast.messages.prompt_declined;
+        const msg = scope.broadcast.msgDeclined;
         if (!msg) {
-          const logMsg = 'undefined broadcast.messages.prompt_declined';
+          const logMsg = 'undefined broadcast.msgDeclined';
```

Comment:
We discussed this in Slack -- going to rename as `declinedMessage` and use that naming convention for all Gambit message properties moving forward.